app/components/QuestionDetail/QuestionDetail.jsx

```diff
@@ -35,57 +34,67 @@ function QuestionDetails(props) {
 
   const { question } = props;
 
-    const initialState = {
-      showAnswerModal: false,
-      showAssignAnswerModal: false,
-    };
-  
+  const initialState = {
+    showAnswerModal: false,
+    showAssignAnswerModal: false,
+  };
+
   const [state, setState] = useState(initialState);
   const [writingCommentOnMobile, setWritingCommentOnMobile] = useState(false);
 
   const { questionId } = useParams();
 
+  const setUrl = () => {
```

Comment:
@AldoDominguezWizeline  What is setUrl used for?

Reply:
I preserved that function from the previous react project, it seems it just replaces the navigation history in the browser from what I could gather, but I tested again and the function is not necessary, so I removed it, nice catch and thank you!